okhttp/src/test/java/okhttp3/OkPutHeadTest.kt

```diff
@@ -0,0 +1,110 @@
+package okhttp3
+
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import okhttp3.testing.Flaky
+import okio.BufferedSink
+import org.junit.Assert.assertEquals
+import org.junit.Assert.fail
+import org.junit.Rule
+import org.junit.Test
+import java.io.IOException
+
+class OkPutHeadTest {
+  @JvmField @Rule val clientTestRule = OkHttpClientTestRule()
+
+  @Test
+  @Throws(Exception::class)
+  @Flaky
```

Comment:
what makes it flaky?

Reply:
I will remove.  It's more that Flaky makes the test print events to the console, so I just wanted the sideeffect.